From e34f1384aedd814bcd35d7126c902479a1e214ad Mon Sep 17 00:00:00 2001
From: Navarum Eliskar <48705414+navarum@users.noreply.github.com>
Date: Tue, 31 Mar 2020 02:44:36 -0700
Subject: [PATCH 01/10] Remove some annoying interactions

- Don't sleep during 'elide'
- Don't ask user about seeing high scores
---
 tetris/input.c  | 14 +++++++-------
 tetris/tetris.c |  8 ++++----
 2 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/tetris/input.c b/tetris/input.c
index 469fa79..dc65980 100644
--- a/tetris/input.c
+++ b/tetris/input.c
@@ -120,14 +120,14 @@ again:
 void
 tsleep()
 {
-	struct timeval tv;
-	char c;
+/* 	struct timeval tv; */
+/* 	char c; */
 
-	tv.tv_sec = 0;
-	tv.tv_usec = fallrate;
-	while (TV_POS(&tv))
-		if (rwait(&tv) && read(0, &c, 1) != 1)
-			break;
+/* 	tv.tv_sec = 0; */
+/* 	tv.tv_usec = fallrate; */
+/* 	while (TV_POS(&tv)) */
+/* 		if (rwait(&tv) && read(0, &c, 1) != 1) */
+/* 			break; */
 }
 
 /*
diff --git a/tetris/tetris.c b/tetris/tetris.c
index ba830bf..3f13ebf 100644
--- a/tetris/tetris.c
+++ b/tetris/tetris.c
@@ -309,11 +309,11 @@ main(argc, argv)
 	    score, score == 1 ? "" : "s", level, score * level);
 	savescore(level);
 
-	printf("\nHit RETURN to see high scores, ^C to skip.\n");
+/* 	printf("\nHit RETURN to see high scores, ^C to skip.\n"); */
 
-	while ((i = getchar()) != '\n')
-		if (i == EOF)
-			break;
+/* 	while ((i = getchar()) != '\n') */
+/* 		if (i == EOF) */
+/* 			break; */
 
 	showscores(level);
 
-- 
2.25.1

